### PR TITLE
data:SohamG934

### PR DESCRIPTION
**Pull request by SohamG934**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4562"></a>

### CODE REVIEW

File: data/SohamG934.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	"bio": "**Budding Engineer** and **tech aficionado**. **Travel enthusiast**",
+	"links": [{
+		"name": "Follow me on Twitter",
+		"url": "https://twitter.com/SOHAMG5",
+		"icon": "FaTwitter"
+	}, 
+            {
+		"name": "Follow me on LinkedIn",
+		"url": "https://www.linkedin.com/in/soham-ghadge-17bb18245/",
+		"icon": "FaLinkedIn"
```
Comment (swagataroy30) [inline data/SohamG934.json:14]:
```suggestion
		"icon": "FaLinkedin"
```

**Review by swagataroy30 — COMMENTED** [review]
Fix the typo. Thank You!

**Review by swagataroy30 — COMMENTED** [review]
You have added a linkedin icon, but here you might want to add a Github Icon - `FaGithub`. You can check this https://linkfree.eddiehub.io/icons, for icon reference. Thank You!

File: data/SohamG934.json (hunk 1 above)
Comment (SohamG934) [reply data/SohamG934.json:14]: Thank you for pointing out the error, it was overlooked by me.

File: data/SohamG934.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	}, 
+            {
+		"name": "Follow me on GitHub",
+		"url": "https://www.github.com/SohamG934",
+		"icon": "FaGithub"
+	},
+          {
+		"name": "Follow my blogs on hashnode",
+		"url": "https://www.sohamghadge.hashnode.dev/",
+		"icon": "FaHashnode"
```
Comment (swagataroy30) [inline data/SohamG934.json:24]:
```suggestion
		"icon": "SiHashnode"
```

Comment (swagataroy30) [inline data/SohamG934.json:24]:
```suggestion
		"icon": "SiHashnode"
```

**Review by swagataroy30 — COMMENTED** [review]
Sorry to not have mentioned this earlier. Can you please change the Hashnode icon to `SiHashnode`, as FaHashnode is not available. Thank You!

**Comment by SohamG934** [comment]
Thank you for the suggestion, I have updated the concerned icon

**Review by iamthecloverly — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**